continuous_integration/appveyor/requirements.txt

```diff
@@ -9,8 +9,8 @@
 # of the rackspace folder instead of trying to install from more recent
 # source tarball published on PyPI
 pytest
-numpy==1.9.3
-scipy==0.16.0
+numpy==1.13.0
+scipy==0.19.0
```

Comment:
I changed the versions here to more recent wheels because once the Appveyor tests started running, the following error was raised during test collection: `ValueError: numpy.dtype has the wrong size, try recompiling. Expected 52, got 56`. We could also try the `scipy-1.0.0rc1` wheels that are on `pypi` presently with the anticipation that `scipy-1.0.0` is scheduled for release next week, along with official `numpy` Windows wheels that are already in the wild.